{"uuid":"df3c2940-cadd-47ee-bf7b-ddfca6df9161","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["b6f840dc-bbb9-40ff-943b-964e2340a2d7"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882777840,"stop":1687882777840},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882778038,"stop":1687882778038},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882778119,"stop":1687882778119},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882778157,"stop":1687882778157},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882778163,"stop":1687882778163}],"start":1687882777747,"stop":1687882778165}